core/src/main/java/org/fao/geonet/kernel/mef/Importer.java

```diff
@@ -397,10 +397,14 @@ public void handleInfo(Element info, int index) throws Exception {
                 }
 
 
-                importRecord(uuid, uuidAction, md, schema, index,
-                    source, sourceName, sourceTranslations,
-                    context, metadataIdMap, createDate,
-                    changeDate, groupId, isTemplate);
+                try {
+                    importRecord(uuid, uuidAction, md, schema, index,
+                       source, sourceName, sourceTranslations,
+                       context, metadataIdMap, createDate,
+                       changeDate, groupId, isTemplate);
+                } catch (Exception e) {
```

Comment:
This is a pretty generic "Exception"? is there something specific you can catch?

Reply:
I caught the exception thrown by importRecord which is Exception.  importRecord thow Exception which is not ideal and should probably be fixed in a separate PR.